runtime/tests/tests.rs

```diff
@@ -385,8 +421,7 @@ fn staking_validators_should_receive_equal_transaction_fee_reward() {
 
 			start_era(1);
 			let validator_len = validators.len() as Balance;
-			let (total_staking_reward, _) = reward_validators(&validators);
-			let per_staking_reward = total_staking_reward / validator_len;
+			reward_validators(&validators);
```

Comment:
[out-of-scope] Not for this PR, but something to consider for later @holygits , this test is a little soft when we are awarding all validators the same points.
i.e. it doesn't answer "are all validators awarded the same fee rewards when their points are different?"

Reply:
This will be addressed in the same followup PR

Reply:
Good point, we could  add different points to validators and check the free balance in the future